src/Resources/assets/js/contao-tinymce-bundle.js

```diff
@@ -7,8 +7,12 @@ import 'tinymce/themes/silver';
 class TinyMceBundle {
 
     static init() {
-        let tinyMceFields = document.querySelectorAll('textarea[data-tinymce="1"]'),
-            language = document.querySelector('html').getAttribute('lang');
+        TinyMceBundle.initTinymce();
+        TinyMceBundle.initEventListeners();
+    }
+
+    static initTinymce() {
```

Comment:
Please take care of a consistent writing of TinyMce -> initTiny**M**ce -> same for other methods like setTinymceError -> setTinyMceError